### PR TITLE
BVPS-52: Template Project Setup

### PR DESCRIPTION
**Pull request by ceciliahangzhou**
https://deloittedigital.atlassian.net/browse/BVPS-52

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ 
+  "semi": false,
+  "tabWidth": 2,
```
Comment (hannah-macdonald1) [inline .prettierrc:3]: Do we want tabs or spaces? Just to keep it consistent with styling on the backend

File: package-lock.json
Comment (hannah-macdonald1) [inline package-lock.json:1]: I usually throw the package-lock in a gitignore, should I include it on the be?

File: .prettierrc (hunk 1 above)
Comment (ceciliahangzhou) [reply .prettierrc:3]: Lets use tabs

Comment (ceciliahangzhou) [reply .prettierrc:3]: will change that, thx

Comment (ceciliahangzhou) [reply .prettierrc]: I have change it to 4, please review again

**Review by hannah-macdonald1 — COMMENTED** [review]
Minor comment about the licensing, otherwise it's good 👍

**Review by hannah-macdonald1 — APPROVED** [review]
Will add the small things I need to my PR. This is good 👍

**Pull request merged**